cdap-common/src/main/java/io/cdap/cdap/common/logging/common/UncaughtExceptionHandler.java

```diff
@@ -40,8 +40,7 @@ public void uncaughtException(Thread t, Throwable e) {
     if (stackTrace.length > 0) {
       Logger logger = LoggerFactory.getLogger(stackTrace[0].getClassName());
       logger.debug("Uncaught exception in thread {}", t, e);
-    } else {
```

Comment:
this doesn't result in double logging? 

If it doesn't, that means logger.debug() gets lost? If so, why do it at all?

Reply:
Wasn't sure if we had any logic depending on this behavior; for context, the reason this didn't log previously is because it logs the InvocationTargetException which wasn't set to a debug level, so it wasn't visible to us.

Reply:
Is that something that can be fixed in the logback xml? Making this change seems like it would double log

Reply:
If it was only this exception then it could be fixed by hardcoding the log level; however, I worry other exceptions which don't originate from CDAP will get swallowed as well. I'll go ahead and just remove the original debug message.

Reply:
if we're doing this we can remove the line that gets the stack trace